src/main/scala/GuessWhoGame/Game.scala

```diff
@@ -22,59 +24,71 @@ object Game extends App {
   val char17: Character = Character("roshan", "male", "brown", "blue", wearsGlasses = true, facialHair = false)
   val char18: Character = Character("patricia", "female", "purple", "yellow", wearsGlasses = false, facialHair = false)
   val char19: Character = Character("dave", "male", "none", "yellow", wearsGlasses = false, facialHair = false)
-  val char20: Character = Character("bobbiana", "female", "Brown", "Blue", wearsGlasses = true, facialHair = false)
+  val char20: Character = Character("bobbiana", "female", "brown", "blue", wearsGlasses = true, facialHair = false)
 
+  //would remain the same throughout
   var gameBoard = Board(List(char1, char2, char3, char4, char5, char6, char7, char8, char9, char10, char11, char12, char13, char14, char15, char16, char17, char18, char19, char20))
 
```

Comment:
Another var here. Is there a way to get around this?

Reply:
Ibid. Eliminated.